root/etc/e-smith/templates/etc/shorewall/rules/15IPS

```diff
@@ -0,0 +1,6 @@
+{
+    my $status = $suricata{'status'} || 'enabled';
+    if ($status eq 'enabled') {
+        $OUT.="\n?SECTION ALL\nNFQUEUE(bypass) all+ all+:!127.0.0.1/32 - - - - - - !0x10/0x10\n\n?SECTION NEW\n";
```

Comment:
Firewall sections must be handled inside firewall template.

Also the exclusion should be more like `127.0.0.0/24`.
Moreover, why not exclude all green IP addresses?